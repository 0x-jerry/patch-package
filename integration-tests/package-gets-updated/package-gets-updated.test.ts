import { runIntegrationTest } from "../runIntegrationTest"
runIntegrationTest("package-gets-updated", true)
