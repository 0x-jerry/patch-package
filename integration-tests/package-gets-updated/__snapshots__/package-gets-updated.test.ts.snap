// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test package-gets-updated: fail when the patch was not applied 1`] = `
"SNAPSHOT: fail when the patch was not applied
[1/4] Resolving packages...
[2/4] Fetching packages...
[3/4] Linking dependencies...
[4/4] Building fresh packages...
success Saved lockfile.
success Saved 1 new dependency.
info Direct dependencies
└─ left-pad@1.1.3
info All dependencies
└─ left-pad@1.1.3
$ patch-package
patch-package: Applying patches...
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
END SNAPSHOT"
`;

exports[`Test package-gets-updated: left-pad should contain patch-package 1`] = `
"SNAPSHOT: left-pad should contain patch-package
    // devide \`len\` by 2, ditch the patch-package
END SNAPSHOT"
`;

exports[`Test package-gets-updated: left-pad should still contain patch-package 1`] = `
"SNAPSHOT: left-pad should still contain patch-package
    // devide \`len\` by 2, ditch the patch-package
END SNAPSHOT"
`;

exports[`Test package-gets-updated: warning when the patch was applied but version changed 1`] = `
"SNAPSHOT: warning when the patch was applied but version changed
[1/4] Resolving packages...
[2/4] Fetching packages...
[3/4] Linking dependencies...
[4/4] Building fresh packages...
success Saved lockfile.
success Saved 1 new dependency.
info Direct dependencies
└─ patch-package@6.0.0-5
info All dependencies
└─ patch-package@6.0.0-5
$ patch-package
patch-package: Applying patches...
END SNAPSHOT"
`;
